lexicons/app/bsky/actor/getSuggestions.json

```diff
@@ -28,7 +28,7 @@
               "type": "array",
               "items": {
                 "type": "ref",
-                "ref": "app.bsky.actor.defs#profileView"
+                "ref": "app.bsky.actor.defs#profileViewDetailed"
```

Comment:
One intentional feature of the way `knownFollowers` was added to `ViewerState` is that it's decoupled from whether we serve a `ProfileView` vs. `ProfileViewDetailed`.  This is nice because it means we can sprinkle `knownFollowers` wherever we want it without converting all those areas to detailed views.

How would we feel about instead leaving the lexicons unchanged and just adding `knownFollowers` into the `ViewerState`s served from this endpoint?